lib/CCreatureHandler.h

```diff
@@ -34,6 +34,7 @@ class DLL_LINKAGE CCreature : public CBonusSystemNode
 
 	CreatureID idNumber;
 	TFaction faction;
+	mutable ETerrainType::EETerrainType nativeTerrain;

```

Comment:
this field need serialization, and to prevent desync add additional mutable fileld

Reply:
This field is removed, there's another field in the Stack class